stages/org.osbuild.container-deploy.meta.json

```diff
@@ -27,6 +27,11 @@
           "items": {
             "type": "string"
           }
+        },
+        "remove-signatures": {
```

Comment:
(mostly note to self) We are sadly a bit inconsistent here, we have around 1/3 of our options with "-" and 2/3 with "_" as deliminator. Given that images already uses "remove-signatures" I guess this is the best course of action but at some point we really should standarize on one :)